src/pages/Home.jsx

```diff
@@ -83,8 +96,10 @@ const Home = () => {
 
       <div className="mt-18">
         {loading ? (
-          <div className="flex justify-center items-center mt-4">
-            <Loader />
+          <div className="flex flex-wrap justify-center gap-5 mt-5">
+            <Loader /><Loader /><Loader />
+            <Loader /><Loader /><Loader />
+            <Loader /><Loader /><Loader />
```

Comment:
doesn't seem a good practice , map it as an array(9).

Reply:
ok 
and  any suggestion for create post page default generating image how it look

Reply:
nothing particular , just the image
![this](https://github.com/yuvrxj-afk/PaintBot/assets/63532070/e10aeff4-a263-40c3-ac4b-090cbb1d6cfb)